source/includes/basics.md

```diff
@@ -6,15 +6,21 @@ If you're looking to interact with the Namara API, you've come to the right plac
 
 The Namara API is a REST-based service that accepts and returns `JSON` in most cases (we'll cover [response formats](#formats-pagination-amp-ordering) later). Requests should be made to:
 
-<code>https://:namara_api_host/v0/:endpoint</code>
+<code>https://{NAMARA_API_HOST}/v0/{ENDPOINT}</code>
 
 Unless indicated otherwise, the Namara API host is `api.namara.io`, which will be used throughout the documentation. 
 
 The responses will be outlined for each endpoint.
 
+## Data Set Versions
+
+Our data is updated regularly and automatically, so values in rows and columns may change. The data set version is a snapshot of the data set's structure and properties. When that structure changes, we create a new version of the data set. 
```

Comment:
no comma in first sentence and make it "so _that_ values...". I don't think the last sentence needs a comma either.

Reply:
That changes the meaning of the sentence to something other than what it should be. The grammar here is fine.